TenantFile/Api/Controllers/OrganizerController.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using FirebaseAdmin.Auth;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using TenantFile.Api.Models;
+using TenantFile.Api.Models.Entities;
+
+namespace TenantFile.Api.Controllers
+{
+    [Authorize(Policy = "AdminOnly")]
+    public class OrganizerController : ControllerBase
+    {
+        private readonly IDbContextFactory<TenantFileContext> dbContextFactory;
+
+        public OrganizerController(IDbContextFactory<TenantFileContext> dbContextFactory)
```

Comment:
Why use a dbcontextfactor instead of injecting the tenantfilecontext?

Reply:
Give injecting the context a shot

Reply:
Injection works in the controllers but I didn't try it in the GraphQL because the implementation of the dataloaders seems tied to the PooledDBFactory. I'm not sure there would be a benefit to using both.